src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    public static final String COMA = ",";
+    public static final String FRUIT = "fruit";
+    public static final String QUANTITY = "quantity";
+
+    @Override
+    public String createReport(Storage storage) {
```

Comment:
Storage should be a class field
```suggestion
    public String createReport() {
```